storybook/src/elements/widgets/widget.stories.js

```diff
@@ -0,0 +1,112 @@
+import { html } from 'lit-html';
+import { storiesOf } from '@storybook/polymer';
+import { text } from '@storybook/addon-knobs';
+import '@nuxeo/nuxeo-ui-elements/widgets/nuxeo-input.js';
+import '@nuxeo/nuxeo-ui-elements/widgets/nuxeo-textarea.js';
+import '@nuxeo/nuxeo-ui-elements/widgets/nuxeo-date-picker.js';
+import '@nuxeo/nuxeo-ui-elements/widgets/nuxeo-selectivity.js';
+import '@nuxeo/nuxeo-ui-elements/nuxeo-data-table/iron-data-table.js';
+import '@nuxeo/nuxeo-ui-elements/widgets/nuxeo-directory-radio-group.js';
+import '@nuxeo/nuxeo-ui-elements/widgets/nuxeo-select.js';
+import { cities as CITIES, LIST } from '../../data/lists.data';
+import { DIRECTORY_SUGGESTION_ENTRIES } from '../../data/directory-suggestion.data.js';
+
+const server = window.nuxeo.mock;
+server.respondWith('post', '/api/v1/automation/Directory.SuggestEntries', () => DIRECTORY_SUGGESTION_ENTRIES);
+storiesOf('Widgets', module).add('Vertical Alignment Consistency', () => {
+  const label = text('Label', 'Label');
+  return html`
+    <style>
+      .container {
+        margin: 2rem;
+      }
+      .row {
+        display: flex;
+        justify-content: space-between;
+      }
+      .row > * {
+        width: 32%;
+      }
+    </style>
+    <div class="container">
+      <div class="row">
+        <nuxeo-input label="${label}" placeholder="Placeholder"></nuxeo-input>
```

Comment:
nit: a knob for the placeholder could be useful to see how it changes across widgets